OracleDataIntegrator/dockerfiles/12.2.1.2.6/Dockerfile

```diff
@@ -0,0 +1,91 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 2016-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Data Integrator 12.2.1.2.6
+#
+# Base image of this dockerfile is Oracle Java 8 docker image.
+#
+# Pre-requisite to build oracle Oracle Data Integrator 12.2.1.2.6 image is to have
+# Oracle Java 8 docker image
+#
+# To build Oracle Java 8 docker image, kindly refer below link
+# https://github.com/oracle/docker-images/tree/master/OracleJava
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fmw_12.2.1.2.6_odi_Disk1_1of2.zip
+# (2) fmw_12.2.1.2.6_odi_Disk1_2of2.zip
+#   Download the production distribution for Oracle Data Integrator from the Oracle Software Delivery Cloud
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Make sure that Oracle Java 8 image is available
+# Run:
+#      $ docker build -t oracle/odi:12.2.1.2.6 .
+#
+#
+# Pull base image
+# ---------------
+FROM oracle/serverjre:8
+
+#
+# Maintainer
+# ----------
+MAINTAINER Manish Vaishnani <manish.vaishnani@oracle.com>
+
+#
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+ENV ODI_PKG1=fmw_12.2.1.2.6_odi_Disk1_1of2.zip \
+    ODI_PKG2=fmw_12.2.1.2.6_odi_Disk1_2of2.zip \
+    ODI_JAR=fmw_12.2.1.2.6_odi.jar \
+    ODI_JAR2=fmw_12.2.1.2.6_odi2.jar \
+    ORACLE_HOME=/u01/oracle \
+    PATH=$PATH:$JAVA_HOME/bin:$ORACLE_HOME/oracle_common/common/bin:$ORACLE_HOME/container-scripts \
```

Comment:
`$JAVA_HOME/bin` should not be needed because the base `oracle/serverjre:8` image uses the `alternatives` system to add the `java`, `javac` and `jar` binaries to `/usr/bin`.